src/ec/packages/ec-component-card/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ecl-twig/ec-component-card",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Card",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
```

Comment:
please add a devDependency to ECL vanilla component ("@ecl/ec-component-card": "^2.1.1")

Reply:
Added in https://github.com/ec-europa/ecl-twig/pull/21/commits/78442bbb33e4a3550799ccb38601a6fc25482e3b

Reply:
I updated the package.json file in the ec-components folder. I entered all current modules, can it remain that way? Should I have only the current module in dependencies? In the next components, I just have to enter the next - current - component over which I work?